app/src/main/java/tec/ac/cr/marape/app/AddGuestActivity.kt

```diff
@@ -1,11 +1,107 @@
 package tec.ac.cr.marape.app
 
+import android.app.AlertDialog
+import android.content.ContentValues.TAG
+import android.content.Context
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
+import android.widget.EditText
+import androidx.core.widget.addTextChangedListener
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import tec.ac.cr.marape.app.adapter.UserView
+import tec.ac.cr.marape.app.model.Inventory
+import tec.ac.cr.marape.app.model.User
 
 class AddGuestActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_add_guest)
+  private val currentUserEmail: String = FirebaseAuth.getInstance().currentUser?.email ?: ""
+  private lateinit var userList: MutableList<User>
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_add_guest)
+
+    val inventory = intent.getStringExtra("idInventory")
+    val searchUser: EditText = findViewById(R.id.entry_user_search);
+    val recyclerView: RecyclerView = findViewById(R.id.recycle_users);
+    recyclerView.layoutManager = LinearLayoutManager(this);
+
+    if(inventory != null){
+      userList = getUsers(inventory);
+      val adapter = UserView(userList, inventory)
+      recyclerView.adapter = adapter
+
+      searchUser.addTextChangedListener { searchText ->
+        val query = searchText.toString().lowercase().trim();
+
+        if(userList.isNotEmpty()){
```

Comment:
Darle formato al código.
```suggestion
        if (userList.isNotEmpty()) {
```